docs/releases/5.0.txt

```diff
@@ -170,6 +170,9 @@ Minor features
   function returns a 2-dimensional point on the geometry that is closest to
   another geometry.
 
+* Added support for filter argument on
+  :doc:`GIS aggregates </ref/contrib/gis/geoquerysets>`.
```

Comment:
```suggestion
* :ref:`GIS aggregates <gis-aggregation-functions>` now support the ``filter``
   argument.
```

Reply:
Thanks I couldn't figure out how to link to a particular  Heading in the document